llvm/include/llvm/Transforms/AvmusatovAS/AvmusatovAS.h

```diff
@@ -0,0 +1,27 @@
+#ifndef LLVM_TRANSFORMS_AVMUSATOVAS_AVMUSATOVAS_H
+#define LLVM_TRANSFORMS_AVMUSATOVAS_AVMUSATOVAS_H
+
+#define DEBUG_TYPE "AvmusatovAS"
```

Comment:
@avmusatov, please remove this macro definition since it's not used for this pass and causes possible redefinition warning

Reply:
@geexie, this line was removed